ga/latest/full/Dockerfile.ubi9-minimal.ibmjava8

```diff
@@ -0,0 +1,51 @@
+# (C) Copyright IBM Corporation 2025.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG PARENT_IMAGE=icr.io/appcafe/websphere-liberty:kernel-java8-ibmjava-ubi9-minimal
+FROM $PARENT_IMAGE AS installBundle
+
+ARG VERBOSE=false
+ARG REPOSITORIES_PROPERTIES=""
+
+# If there is a local copy of the repository use that instead
+COPY resources/ /tmp/
+
+# Install the base bundle
+RUN set -eux; \
+  if [ ! -z "$REPOSITORIES_PROPERTIES" ]; then \
+    mkdir /opt/ibm/wlp/etc/; \
+    echo "$REPOSITORIES_PROPERTIES" > /opt/ibm/wlp/etc/repositories.properties; \
+    installUtility install --acceptLicense baseBundle; \
+    rm /opt/ibm/wlp/etc/repositories.properties; \
+  elif [ -f /tmp/wlpRepo.zip ]; then \
+    installUtility install --acceptLicense baseBundle --from=/tmp/wlpRepo.zip; \
+  else \
+    installUtility install --acceptLicense baseBundle; \
+  fi; \
+  rm -rf /output/workarea /output/logs; \
+  find /opt/ibm/wlp ! -perm -g=rw -print0 | xargs -r -0 chmod g+rw;
+
+ARG PARENT_IMAGE=icr.io/appcafe/websphere-liberty:kernel-java8-ibmjava-ubi9-minimal
+FROM $PARENT_IMAGE
+ARG VERBOSE=false
+
+# Copy the runtime
+COPY --from=installBundle --chown=1001:0 /opt/ibm/wlp /opt/ibm/wlp
+
+COPY --chown=1001:0 server.xml /config/
+
+# Create a new SCC layer
+RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
+    && rm -rf /output/messaging /output/resources/security /logs/* $WLP_OUTPUT_DIR/.classCache \
+    && find /opt/ibm/wlp/output ! -perm -g=rwx -print0 | xargs -0 -r chmod g+rwx
```

Comment:
`-path "*.classCache*"` should be added to match IBM Java 8 + UBI 8: https://github.com/WASdev/ci.docker/blob/83ed16070f361e1e83efee1bf5a505947d16c3bc/ga/latest/full/Dockerfile.ubi.ibmjava8#L51